cc.cubocore.CoreTime.appdata.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2021 CuboCore -->
+
+<component type="desktop-application">
+    <provides>
+        <id>cc.cubocore.CoreTime</id>
+    </provides>
+    <replaces>
+        <id>org.cubocore.CoreTime</id>
+    </replaces>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-3.0</project_license>
+
+    <project_group>CuboCore</project_group>
+    <developer_name>CuboCore Group</developer_name>
+
+    <launchable type="desktop-id">cc.cubocore.CoreTime.desktop</launchable>
+
+    <name>CoreTime</name>
+    <summary>A time related task manager</summary>
+
+    <description>
+        <p>
+            CoreTime is a very simple Clock/Calendar application which shows a clock, supports
+            alarms, reminders, sports a stop watch and a timer.
+        </p>
+    </description>
+
+    <keywords>
+        <keyword>CoreTime</keyword>
+        <keyword>Clock</keyword>
+        <keyword>Calendar</keyword>
+        <keyword>Stopwatch</keyword>
+        <keyword>Alarm</keyword>
+        <keyword>Reminder</keyword>
+        <keyword>CoreApps</keyword>
+        <keyword>CSuite</keyword>
+        <keyword>CuboCore</keyword>
+    </keywords>
+
+    <categories>
+        <category>Utility</category>
+        <category>Qt</category>
+        <category>X-CSuite</category>
+    </categories>
+
+    <content_rating type="oars-1.1" />
+    <url type="homepage">https://cubocore.gitlab.io/</url>
+    <url type="help">https://gitlab.com/cubocore/coreapps/coretime/-/blob/master/README.md</url>
+    <url type="bugtracker">https://gitlab.com/cubocore/coreapps/coretime/-/issues</url>
+    <url type="contact">https://app.element.io/#/room/#cubocore:matrix.org</url>
+
+    <releases>
+        <release version="5.0.0" date="2025-03-9">
+            <url type="details">https://gitlab.com/cubocore/wiki/-/blob/master/ChangeLog</url>
+            <description>
+                <p>Release description</p>
+                <ul>
+                    <li>Domain changed from .Org to .CC.</li>
+                    <li>All apps and libs are now ported and only works with Qt6.</li>
+                </ul>
+            </description>
+        </release>
+        <release version="4.5.0" date="2023-08-27" />
+        <release version="4.4.0" date="2022-08-02" />
+        <release version="4.3.0" date="2021-12-25" />
+    </releases>
+
+    <screenshots>
+        <screenshot type="default">
+            <caption>CoreTime</caption>
+            <image type="source">
+                https://gitlab.com/cubocore/coreapps/coretime/-/raw/master/coretime.png</image>
```

Comment:
```suggestion
            <image type="source">https://gitlab.com/cubocore/coreapps/coretime/-/raw/68ca8b0922afec80a31e0d8d269cc5599eed265a/coretime.png</image>
```

Reply:
pls can you explain, what's the difference?